index.js

```diff
@@ -0,0 +1,46 @@
+import displayLibrary from './modules/display-books.js';
+import manipulateBooks from './modules/edit-library.js';
+import navigation from './modules/navigations.js';
+import booksData from './modules/books-data.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+const addButton = document.querySelector('#add');
+const listContainer = document.querySelector('.books');
+const list = document.querySelector('#list');
+const goAddBook = document.querySelector('#add-book');
+const contact = document.querySelector('#contact');
+const date = document.querySelector('#date');
+
+function displayDate() {
+  const dt = DateTime.now();
+  date.innerHTML = dt.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}
```

Comment:
- [ ] Geat job for the displayDate() function implementation, :clap: but I think it will be better if you can use arrow syntax for functions insted of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
This issue is well resolved! ✔️